addons/website_sale/views/templates.xml

```diff
@@ -1178,13 +1180,13 @@
                        "display_currency": product.currency_id,
                 }'/>
                 <t t-if="is_view_active('website_sale.tax_indication')" t-call="website_sale.tax_indication"/>
-                <del t-if="product.compare_list_price">
+                <del t-if="combination_info['compare_list_price'] and (combination_info['compare_list_price'] &gt; combination_info['price'])">
                     <bdi dir="inherit">
                     <span t-field="product.compare_list_price"
                         groups="website_sale.group_product_price_comparison"
                         t-options='{
                             "widget": "monetary",
-                            "display_currency": product.currency_id,
+                            "display_currency": website.currency_id,
```

Comment:
This is strange IMHO.
AFAIK, t-field should display the record field value as is, without any currency conversion (unless the widget is given a `from_currency` option).
So here, you would display the comparison_price of the product (specified in `product.currency_id`), but show it in the website currency, without any conversion.

I know it's like that in master (since #87134), but I doubt it's a correct change.
Is the change on this line necessary for the bug of the client ?